pkg/build/util/util.go

```diff
@@ -9,8 +9,12 @@ import (
 	"github.com/openshift/origin/pkg/util/namer"
 )
 
-// BuildPodSuffix is the suffix used to append to a build pod name given a build name
-const BuildPodSuffix = "build"
+const (
+	// BuildPodSuffix is the suffix used to append to a build pod name given a build name
+	BuildPodSuffix = "build"
+	// NoBuildLogsMessage reports that no build logs are available
+	NoBuildLogsMessage = "No logs are available"
```

Comment:
Someplace build specific, like build/API.  Pulling out string messages is
usually bad, because people should not be using that string in other code
to check, and our API messages are usually in a different voice than our
cli messages.

On Jun 29, 2015, at 11:09 PM, Ben Parees <notifications@github.com> wrote:

In pkg/build/util/util.go
<https://github.com/openshift/origin/pull/3449#discussion_r33536926>:

> @@ -9,8 +9,12 @@ import (
>  	"github.com/openshift/origin/pkg/util/namer"
>  )
>
> -// BuildPodSuffix is the suffix used to append to a build pod name given a build name
> -const BuildPodSuffix = "build"
> +const (
> +	// BuildPodSuffix is the suffix used to append to a build pod name given a build name
> +	BuildPodSuffix = "build"
> +	// NoBuildLogsMessage reports that no build logs are available
> +	NoBuildLogsMessage = "No logs are available"

it is a bit odd but given that the string is used in at least two places I
can live with it.. is there somewhere else you'd recommend storing strings
like this?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/3449/files#r33536926>.